indexer/src/main/kotlin/com/openlattice/indexing/BackgroundIndexedEntitiesDeletionService.kt

```diff
@@ -187,4 +245,10 @@ class BackgroundIndexedEntitiesDeletionService(
     private fun updateExpiration(entitySetId: UUID) {
         deletionLocks.set(entitySetId, System.currentTimeMillis() + DELETE_EXPIRATION_MILLIS)
     }
+
+    data class EntitySetForDeletion(
+            val id: UUID,
+            val name: String,
+            val partitions: Set<Int>
+    )
```

Comment:
This should be in its own file.

Reply:
nope actually since it's a data class that exists solely within this class solely as a helper for extracting certain features and doesn't have any intrinsic meaning, it should exist only in this class.